OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -990,4 +990,4 @@
     <Message Text="DEBUG TargetName:             $(TargetName)"/>
     -->
   </Target>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Style nit: all our editors fight over formatting, so our convention is to only commit the essential project file changes in each PR.

Reply:
Fixed.